lambda_src/lambda_function.py

```diff
@@ -146,9 +147,63 @@ def sync_flow(event: Any, context: Any = None) -> Dict[Text, Any]:
                 row_result,
             ]
         )
+    return res_data
+
+
+def sync_flow(event: Any, context: Any = None) -> Optional[ResponseType]:
+    """
+    Handles the synchronous part of the generic lambda flows.
+
+    Args:
+        event (Any): This the event object as received by the lambda_handler()
+        context (Any): Has the function context. Defaults to None.
+
+    Returns:
+        Optional[ResponseType]: Represents the response status and data.
+    """
+    LOG.debug('Destination header not found in a POST and hence using sync_flow().')
+    headers = event['headers']
+    req_body = loads(event['body'])
+    start_time = timer()
+
+    destination_driver = None
+    batch_id = headers[BATCH_ID_HEADER]
+    write_uri = headers.get('write-uri')
+    destination_driver = (
+        import_module(f'geff.drivers.destination_{urlparse(write_uri).scheme}')
+        if write_uri
+        else None
+    )
+
+    LOG.debug(f'sync_flow() received destination: {write_uri}.')
+
+    if BATCH_LOCKING_ENABLED and not destination_driver:
```

Comment:
@sfc-gh-afedorov Would it be better to pass a generic `BATCH_LOCKING_ENABLED` boolean instead of a module specific `DYNAMODB_TABLE`?

Reply:
Yes, and mention it in the interface if it's something all modules should export